suite-native/device-manager/src/components/PortfolioTrackerDeviceManagerContent.tsx

```diff
@@ -32,20 +39,42 @@ export const PortfolioTrackerDeviceManagerContent = () => {
         });
     };
 
+    const handleOpenEduLink = () => {
+        openLink('https://trezor.io/learn');
```

Comment:
Doesn't we have these link in more than one place? Maybe create some const for it?

Reply:
We have multiple links that starts with `https://trezor.io/learn` prefix, but there is no other place where is the URL just the same as here (all the other cases redirects to some specific subpage of `learn`). So I think there is no need for const now. Or what do you think?